docs/components/iam/deployment/configuration-variables.md

```diff
@@ -0,0 +1,30 @@
+---
+id: configuration-variables
+title: "Configuration variables"
+sidebar_label: "Configuration variables"
+---
+
+### Feature control
+
+Name | Description | Default value
+-----|-------------|--------------
+feature.ldap | Toggle LDAP support within IAM | false
```

Comment:
It should be mentioned how these variables are mapped to environment variables or just directly state them instead of application properties format

`FEATURE_LDAP`